duplex_http_call.go

```diff
@@ -248,7 +255,8 @@ func (d *duplexHTTPCall) makeRequest() {
 	if err := d.validateResponse(response); err != nil {
 		d.SetError(err)
 		return
-	} else if (d.streamType&StreamTypeBidi) == StreamTypeBidi && response.ProtoMajor < 2 {
+	}
+	if (d.streamType&StreamTypeBidi) == StreamTypeBidi && response.ProtoMajor < 2 {
```

Comment:
Small style thing here (no else after return). I'm happy to revert though if we're ok with this style.